ocp_resources/catalog_source_config.py

```diff
@@ -69,7 +69,7 @@ def wait_for_csc_status(self, status, timeout=120):
         samples = TimeoutSampler(
             wait_timeout=timeout,
             sleep=1,
-            exceptions=ProtocolError,
+            exceptions_dict={ProtocolError: []},
```

Comment:
Please save `{ProtocolError: []}` and re-use in all places

Reply:
done.

Reply:
can you please upload a new patchset with the fix?

Reply:
i already uploaded a commit (6b104a3) before commenting. what was the problem?